cmd/keycmd/fund.go

```diff
@@ -0,0 +1,289 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package keycmd
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/key"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/genesis"
+	"github.com/ava-labs/avalanchego/ids"
+	avago_constants "github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/crypto/keychain"
+	ledger "github.com/ava-labs/avalanchego/utils/crypto/ledger"
+	"github.com/ava-labs/avalanchego/utils/formatting/address"
+	"github.com/ava-labs/avalanchego/utils/units"
+	"github.com/ava-labs/avalanchego/vms/components/avax"
+	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
+)
+
+const (
+	sourceFlag          = "source"
+	targetFlag          = "target"
+	keyNameFlag         = "key"
+	ledgerIndexFlag     = "ledger"
+	wrongLedgerIndexVal = 32768
+)
+
+var (
+	source      bool
+	target      bool
+	keyName     string
+	ledgerIndex uint
+)
+
+func newFundCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "fund addr amount",
```

Comment:
Normally, we would prompt for the address and amount, I don't think there's a reason to break with this convention here

Reply:
`transfer` might be a better description than `fund`